filebeat/module/elasticsearch/server/test/test-json.log-expected.json

```diff
@@ -6,7 +6,7 @@
         "elasticsearch.node.name": "node-0",
         "event.dataset": "elasticsearch.server",
         "event.module": "elasticsearch",
-        "event.timezone": "+00:00",
+        "event.timezone": "-02:00",
```

Comment:
Similar to my other comment: https://github.com/elastic/beats/pull/13874/files#r330526232.